.github/workflows/ci.yml

```diff
@@ -46,13 +46,31 @@ jobs:
           # latest dev build
           # https://www.sublimetext.com/dev
           - sublime-channel: dev
-            sublime-build: 4134
+            sublime-build: 4137
             optional: false
 
     steps:
 
+      - name: Git config (global)
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          git config --global init.defaultBranch master
+
+      # https://github.com/actions/checkout
       - name: Git checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
+
+      - name: Clean up for automated testing
+        run: |
+          # remove (some) subdirectories in root directory
+          find . -maxdepth 1 -mindepth 1 -type d -name '.git' -exec rm -rf '{}' ';'
+          find . -maxdepth 1 -mindepth 1 -type d -name '.github' -exec rm -rf '{}' ';'
+          # remove (some) files in root directory
+          find . -maxdepth 1 -mindepth 1 -type f -name '.gitignore' -delete
+          find . -maxdepth 1 -mindepth 1 -type f -name 'LICENSE' -delete
+          find . -maxdepth 1 -mindepth 1 -type f -name 'README.md' -delete
+        shell: bash
```

Comment:
What's the benefit of cleaning up test environment?

Reply:
As the syntax test binary just looks for `syntax_test_...` files, I don't think that kind of extra work is needed. File traversal itself is fast enough maybe even faster than looking for and removing those files first.